dapps/dapps_dev.json

```diff
@@ -756,6 +756,16 @@
       ],
       "desktopOnly": true
     },
+    {
+      "name": "Big Ballz of Bajun (Desktop mode)",
+      "icon": "https://raw.githubusercontent.com/novasamatech/nova-utils/master/icons/dapps/color/BigBallzOfBajun.png",
+      "url": "https://bbb.ajuna.io/",
+      "categories": [
+        "nft",
+        "gaming"
+      ],
+      "desktopOnly": true
```

Comment:
have you added this option for some purpose? cause it's opening dapp in desktop view and is implemented for dapps, that are not mobile friendly. As far as I see dapp is functioning normally in mobile view, so maybe we can remove it?

Reply:
Thx, for pointing that out, yes, we set it to false. @leohar